homework/Roman/task_05/task_05.py

```diff
@@ -0,0 +1,20 @@
+def sort_odd(in_data):
+    odd_vars = []
+    loc_in_data = in_data
+
+    for i in loc_in_data:
+        if int(i) % 2 != 0:
+            odd_vars.append(i)
+
+    odd_vars.sort()
+
+    cnt = 0
+    j = 0
+    for i in loc_in_data:
```

Comment:
```
    for inx, elem in enumerate(loc_in_data):
        if int(elem) % 2 != 0:
            loc_in_data[j] = odd_vars.pop(0)
```
we can do this without extra variables

Reply:
also we can use `map()` function to overlay them

Reply:
Tried both variants.